### PR TITLE
ref(metrics): Make visualize field wider

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Before:**

https://github.com/user-attachments/assets/dd0fa41b-bb87-4c56-8bd5-4f47822e452d

**After:**

https://github.com/user-attachments/assets/2bb7e432-78fb-4482-ba1d-c24432c2c44c

### CODE REVIEW

**Comment by vgrozdanic** [comment]
When you select very long metric name, currently this box would re-adjust and become larger, with this changes, that is no longer the case, and full metric name can't fit in the box anymore (metric name from the screenshot is: allocation_policy.db_request_rejected)
image

File: static/app/components/metrics/queryBuilder.tsx [hunk 1]
```diff
@@ -216,13 +216,45 @@ export const QueryBuilder = memo(function QueryBuilder({
...
-              <QueryFieldGroup>
-                <QueryFieldGroup.Label>{t('Visualize')}</QueryFieldGroup.Label>
+        {!hasMetricsNewInputs(organization) && (
+          <FlexBlock>
+            <GuideAnchor
+              target="metrics_selector"
+              position="bottom"
+              disabled={index !== 0}
+            >
+              {hasMetricsNewInputs(organization) ? (
```
Comment (vgrozdanic) [inline static/app/components/metrics/queryBuilder.tsx:244]: This check is redundant, it will always be false, because of the check on line 237

Comment (priscilawebdev) [reply static/app/components/metrics/queryBuilder.tsx:244]: oh yes! good catch

**Comment by vgrozdanic** [comment]
There is still regression from the current state:
image

new state:
image

The metric name still has 3 dots, but it is not shown the same way as it was before

**Comment by priscilawebdev** [comment]
@vgrozdanic thanks for the great feedback! This issue only happens on very small viewports and the ellipsis you're referencing isn't coming from the styles, but we do trim the string if it's too long via the function https://github.com/getsentry/sentry/blob/0067c2809f76a9dc9b3885d3181cb42b909e4740/static/app/utils/string/middleEllipsis.tsx#L15 . I can create a separate issue, so we can allocate some time to fix it, but merging this PR is fine. This is not a regression, in the old experience we did not respect the max-width of the parent element and the content is overflowing

**Review by matejminar — APPROVED** [review]

**Pull request merged**